sheetstorm/management/commands/generate_test_data.py

```diff
@@ -43,21 +47,21 @@ def handle(self, *args, **options):
         logging.info(f"Total number of projects in the database: {Project.objects.count()}")
 
     @staticmethod
-    def _get_user_options(options):
+    def _get_user_options(options: Dict[str, Union[bool, int, None]]) -> Dict[str, Optional[int]]:
```

Comment:
Now when I'm looking at name "options" I think it is too general, but it is not for now. You will need to fix it in some next PRs

Reply:
Also if you're using multiple times the same typing like `Dict[str, Union[bool, int, None]]`  you can extract it to separate well named variable and use instead of whole `Dict[str, Union[bool, int, None]]`